extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -44,6 +44,7 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverA9VideoBidderCountries';
 		$vars[] = 'wgAdDriverDelayCountries';
 		$vars[] = 'wgAdDriverDelayTimeout';
+		$vars[] = 'wgAdDriverLoadMedrecFromBTF';
```

Comment:
Please add `Countries` to variable name so it's more descriptive

Reply:
Removed in this branch